run_tests.sh

```
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -eou pipefail
+
+# given
+export $(cat .env.example | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)